deepfence_server/pkg/registry/acr/client.go

```diff
@@ -11,39 +12,103 @@ import (
 	"github.com/deepfence/ThreatMapper/deepfence_utils/log"
 )
 
-var client = &http.Client{Timeout: 10 * time.Second}
+const (
+	PerPageCount       = 100
+	ParallelImageFetch = 10
```

Comment:
Should we keep a low and high threshold for the number of workers?
Reason being, if we have less number of repositories, we can work with less number of workers. 

In the init, we create the number of workers = low threshold. 
Lets say we want to allocate 10 repositories per worker. So we can calculate the number of workers using `num_repo/10`. If this count is > 10, we limit the number to high threshold of 10.